certora/specs/SafeMath.spec

```diff
@@ -1,33 +1,32 @@
 // SPDX-License-Identifier: GPL-2.0-or-later
 
-definition WAD() returns uint256 = 10^18;
+definition WAD() returns mathint = 10^18;
 
-function summaryWMulDown(uint256 x,uint256 y) returns uint256 {
-    // Safe require because the reference implementation would revert.
-    return require_uint256((x * y)/WAD());
+function summaryWMulDown(mathint x,mathint y) returns mathint {
+    return (x * y) / WAD();
 }
 
-function summaryWDivUp(uint256 x,uint256 y) returns uint256 {
-    // Safe require because the reference implementation would revert.
-    return require_uint256((x * WAD() + (y-1)) / y);
+function summaryWDivUp(mathint x,mathint y) returns mathint {
+    return x * WAD() + (y-1) / y;
```

Comment:
```suggestion
    return (x * WAD() + (y-1)) / y;
```

Reply:
Fixed in https://github.com/morpho-org/pre-liquidation/pull/89/commits/c715feb778826566a40deb4543be216a1c98523e.